core/src/main/resources/libs/howlite/components/title/dialog/.content.json

```diff
@@ -20,6 +20,12 @@
         "label": "Heading text",
         "required": true
       },
+      "anchorId": {
+        "sling:resourceType": "wcm/dialogs/components/textfield",
+        "name": "anchorId",
+        "default": "Add your #anchorId here",
```

Comment:
I think we have no "default" property defined for textfield? But even if I'm wrong we should not define the default value for this field.

Reply:
fixed